src/main/java/io/appform/dropwizard/actors/ConnectionRegistry.java

```diff
@@ -43,6 +44,12 @@ public RMQConnection createOrGet(String connectionName) {
     }
 
     public RMQConnection createOrGet(String connectionName, int threadPoolSize) {
+
+        if (Constants.DEFAULT_CONNECTIONS.contains(connectionName)) {
+            throw new RabbitmqActorException(ErrorCode.CONNECTION_NAME_RESERVED_FOR_INTERNAL_USE,
+                    "Please don't use default connection names", null);
```

Comment:
As part of log line, let's print all the reserved connection names. In the absence of this, users of library would have to comb through code to figure out all reserved names

Reply:
done